src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -98,6 +98,90 @@ paths:
         httpMethod: GET
 
 
+  #
+  # Temporary Certificate
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/temporary-certificate':
+    parameters:
+      - name: eroId
+        description: The ID of the Electoral Registration Office responsible for the application.
+        schema:
+          type: string
+        in: path
+        required: true
+    options:
+      summary: CORS support
+      description: |
+        Enable CORS by returning correct headers
+      tags:
+        - Temporary Certificate
+      responses:
+        200:
+          description: Default response for CORS method
+          headers:
+            Access-Control-Allow-Origin:
+              schema:
+                type: string
+            Access-Control-Allow-Methods:
+              schema:
+                type: string
+            Access-Control-Allow-Headers:
+              schema:
+                type: string
+          content: { }
+      x-amazon-apigateway-integration:
+        type: mock
+        requestTemplates:
+          application/json: |
+            {
+              "statusCode" : 200
+            }
+        responses:
+          default:
+            statusCode: "200"
+            responseParameters:
+              method.response.header.Access-Control-Allow-Headers: '''Content-Type,X-Amz-Date,Authorization,X-Api-Key'''
+              method.response.header.Access-Control-Allow-Methods: '''*'''
+              method.response.header.Access-Control-Allow-Origin: '''*'''
+            responseTemplates:
+              application/json: |
+                {}
+    post:
+      summary: Generates and returns a temporary certificate
+      description: Generates and returns a temporary certificate
+      tags:
+        - Temporary Certificate
+      requestBody:
+        $ref: '#/components/requestBodies/GenerateTemporaryCertificate'
+      responses:
+        '201':
+          description: created
+          headers:
+            Content-Disposition:
+              description: Content-Disposition header
+              schema:
+                type: string
+                example: attachment; filename="temporary-certificate-V3JSZC4CRH.pdf"
```

Comment:
As multiple temporary certificates can be created for same application reference, possible including the certificate number in the filename is better.

Reply:
Good idea; yes, I'll do that 👍